### PR TITLE
Anax clitroubleshootupdate cli

### PR DESCRIPTION
**Pull request by anaxceron**
Signed-off-by: anaxceron <ana.ceron@broadcom.com>

- Documented new and existing commands that can be used in troubleshooting/info gathering for Zowe CLI
- Created new page "Using individual commands for Zowe CLI troubleshooting" for existing doc
- Updated [Troubleshooting Zowe CLI](https://docs.zowe.org/stable/troubleshoot/cli/troubleshoot-cli) to include separate mentions of new doc and existing-but-restructured doc

Once approved, will be merged w/ Docs Staging so content is published w/ 2.5 release

### CODE REVIEW

**Review by gejohnston — CHANGES_REQUESTED** [review]
Everything that you wrote looks fine. I identified some old content that is now out of date and should be changed.

File: docs/troubleshoot/cli/use-individual-troubleshoot-commands.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+## Identify the currently installed CLI version
+
+Issue the following command:
+
+```
+zowe -V
+```
+
+The exact Zowe CLI version may vary depending upon if the `@latest` or `@zowe-v1-lts`, or `@lts-incremental` version is installed.
```
[the remark this answers is not in the document]
Comment (anaxceron) [reply docs/troubleshoot/cli/use-individual-troubleshoot-commands.md:13]: updated per your comment -- thank you!

File: docs/troubleshoot/cli/use-individual-troubleshoot-commands.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+For the `@zowe-v1-lts` and the `@latest` (forward-development) version tags:
```
[the remark this answers is not in the document]
Comment (anaxceron) [reply docs/troubleshoot/cli/use-individual-troubleshoot-commands.md:15]: done!

File: docs/troubleshoot/cli/use-individual-troubleshoot-commands.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+```
+npm list -g @zowe/cli
+```
+
+For the `@lts-incremental` version tag:
+
+```
+npm list -g @brightside/core
+```
```
[the remark this answers is not in the document]
Comment (anaxceron) [reply docs/troubleshoot/cli/use-individual-troubleshoot-commands.md:25]: done!

File: docs/troubleshoot/cli/use-individual-troubleshoot-commands.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+- Imperative CLI Framework log, which generally contains installation and configuration information.
+
+- Zowe CLI log, which contains information about interaction between CLI and the server endpoints.
+
+Analyze these logs for any information relevant to your issue.
+
+### Profile configuration
+
+The `profiles` folder stores connection information.
```
[the remark this answers is not in the document]
Comment (anaxceron) [reply docs/troubleshoot/cli/use-individual-troubleshoot-commands.md:82]: Made the update. Please review after I push the latest draft of this content.

**Comment by anaxceron** [comment]
@gejohnston -- can you take a quick look at the "v2 team config" content just to make sure it's 100 percent correct?

**Review by gejohnston — APPROVED** [review]
Everything looks good to me.

**Pull request merged**